### PR TITLE
examples/bridge-domain updated

### PR DESCRIPTION
**Pull request by uablrek**
* Add non-point-to-point ipam
* Create the bridge and keep it rather than create/destroy on demand
* Code clean-up
* Example fixup and documentation

### CODE REVIEW

**Review by nickolaev — COMMENTED** [review]
Looks good.
@uablrek would you mind trying to enable it in the CI too?

**Review by nickolaev — COMMENTED** [review]
Hmm, how do you deploy the IPv6 example?
I was planning to allow for multi-configs in a single example. Any ideas?

**Comment by uablrek** [comment]
@nickolaev I don't know how.

Ipv6 examplle added.

**Comment by nickolaev** [comment]
@uablrek OK I can add it to CI in a follow up CI as long as you can confirm it worked locally for you

**Comment by uablrek** [comment]
Yes. The example in README.md works.

> I was planning to allow for multi-configs in a single example. Any ideas?

You mean add both ipv4 and ipv6 addresses (dual-stack) to the simple-client pods?

I must learn more NSM to do that :smile:

**Comment by nickolaev** [comment]
No, it is more about being able to add flavors to examples:
`make k8s-bridge-ipv4-install`
and
`make k8s-bridge-ipv6-install`

And these targets shall be generated automagically by Make :) I will think a bit and figure something out.

Thanks for the awesome work!

File: examples/bridge-domain/bridge/cmd/composite.go [hunk 1]
```diff
@@ -79,12 +91,24 @@ func newVppAgentBridgeComposite(configuration *common.NSConfiguration) *vppAgent
...
+		Name:                "brd",
+		Flood:               true,
+		UnknownUnicastFlood: true,
+		Forward:             true,
+		Learn:               true,
+		MacAge:              120,
+		Interfaces:          make([]*l2.BridgeDomain_Interface, 0),
+	}
 
 	newVppAgentBridgeComposite := &vppAgentBridgeComposite{
```
Comment (edwarnicke) [inline examples/bridge-domain/bridge/cmd/composite.go:104]: This is excellent work.. would you consider adding it to the SDK so folks could use it as a reusable component for building their own NSEs?  This is quite a bit better than our existing [bridge component](https://github.com/networkservicemesh/networkservicemesh/blob/master/sdk/vppagent/bridge.go) in the SDK :)

**Pull request merged**